src/components/CornerDialog/CornerDialog.styles.ts

```diff
@@ -0,0 +1,85 @@
+import type { BaseProps } from '@/types/BaseProps';
+
+export type CornerDialogFooterConfig = {
+  actions?: BaseProps;
+} & BaseProps;
+
+export type CornerDialogConfig = BaseProps & {
+  innerElements?: {
+    intentIndicator?: BaseProps;
+    intentWarning?: BaseProps;
+    intentNegative?: BaseProps;
+    body?: BaseProps;
+    header?: BaseProps;
+    headerTitle?: BaseProps;
+    closeButton?: BaseProps;
+    content?: BaseProps;
+    footer?: CornerDialogFooterConfig;
+  };
+};
+
+export const defaultConfig = {
+  display: 'flex',
+  w: 'fit-content',
+  minWidth: '420px',
+  p: '$space-component-padding-2xLarge',
+  flexDirection: 'row',
+  alignItems: 'flex-start',
+  gap: '$space-component-padding-large',
+  borderRadius: '$border-radius-xLarge',
+  bg: '$color-interaction-background-modeless',
+  boxShadow: '$elevation-bottom-300',
+  borderWidth: '$border-width-small',
+  borderStyle: '$border-style-solid',
+  borderColor: '$color-border-defaultA',
+  overflow: 'hidden',
+  innerElements: {
+    intentIndicator: {
+      h: '$size-xSmall',
+      display: 'flex',
+      alignItems: 'flex-end',
+    },
+    intentWarning: {
+      color: '$color-orange-0',
```

Comment:
```suggestion
      color: '$color-content-warning-secondary',
```
It's really important to set up correct tokens, we have muli level aliasin

<img width="967" alt="Zrzut ekranu 2024-04-25 o 16 54 51" src="https://github.com/VirtusLab/tetrisly-react/assets/6291752/b521e768-cbfb-47aa-b8eb-71d3bb212d35">
g there.